integrationTests/mock/vmContainerMock.go

```diff
@@ -71,8 +71,5 @@ func (V *VMContainerMock) Keys() [][]byte {
 
 // IsInterfaceNil returns true if there is no value under the interface
 func (V *VMContainerMock) IsInterfaceNil() bool {
-	if V == nil {
-		return true
-	}
-	return false
+	return V == nil
```

Comment:
please rename the receiver to vmc in all methods for this mock

Reply:
done.